src/modules/interactive.js

```diff
@@ -0,0 +1,20 @@
+const todoList = document.querySelector('.todo-list');
+
+export default function updateTodos(newTodos) {
+  const updatedTodos = [];
+  for (let i = 0; i < newTodos.length; i += 1) {
+    updatedTodos.push({ ...newTodos[i], id: i + 1 });
+  }
```

Comment:
You could use the map method instead of a for loop to transform the newTodos array into an array of updated todo objects with correct IDs.